internal/receiver/smartagentreceiver/receiver_test.go

```diff
@@ -322,6 +323,23 @@ func TestSmartAgentConfigProviderOverrides(t *testing.T) {
 	require.Equal(t, filepath.Join("/opt", "jre"), os.Getenv("JAVA_HOME"))
 }
 
+func TestReceiverLifecycle(t *testing.T) {
```

Comment:
There's already a few tests like this (`TestMultipleStartAndShutdownInvocations`).  Mind moving up near them and renaming for the exercised behavior?  ~`TestMultipleInstacesOfSameMonitorType`

Reply:
Done via https://github.com/signalfx/splunk-otel-collector/pull/166/commits/2b856ff7ac33f87a6541a8410b57addb6618316e